src/main/java/org/sonatype/nexus/ci/iq/SelectedApplication.groovy

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.iq
+
+import org.sonatype.nexus.ci.util.IqUtil
+
+import hudson.Extension
+import hudson.model.Job
+import hudson.util.ListBoxModel
+import org.kohsuke.stapler.AncestorInPath
+import org.kohsuke.stapler.DataBoundConstructor
+import org.kohsuke.stapler.QueryParameter
+
+class SelectedApplication
+    extends IqApplication
+{
+  @DataBoundConstructor
+  SelectedApplication(final String applicationId) {
+    super(applicationId)
+  }
+
+  @Extension
+  static class DescriptorImpl
```

Comment:
Currently the pipeline syntax generator spits out this:
```
nexusPolicyEvaluation failBuildOnNetworkError: false, iqApplication: [$class: 'SelectedApplication', applicationId: ''], iqStage: '', jobCredentialsId: ''
```
The Symbol annotation should be used to make this a bit cleaner.

Reply:
> The Symbol annotation should be used to make this a bit cleaner.

Change 32b411f